vaje.tex

```diff
@@ -11,10 +11,14 @@
 \usepackage{amsmath}
 \usepackage{amssymb}
 \usepackage{amsthm}
+\usepackage{amsfonts}
 
 % Risanje slik
 \usepackage{tikz}
-
+\usepackage{mathrsfs}%za risanje grafov s pomočjo geogebre
```

Comment:
Je ta package res potreben? Pri meni je bil irelevanten za risanje grafov in sem ga zbrisala.